neuralprophet/time_net.py

```diff
@@ -793,6 +793,9 @@ def test_step(self, batch, batch_idx):
         loss, reg_loss = self.loss_func(inputs, predicted, targets)
         # Metrics
         if self.metrics_enabled:
+            predicted_denorm = self.denormalize(predicted[:, :, 0])
+            target_denorm = self.denormalize(targets.squeeze(dim=2))
+            self.log_dict(self.metrics_val(predicted_denorm, target_denorm), **self.log_args)
             self.log("Loss_test", loss, **self.log_args)
```

Comment:
This is to log (and eventually return) the error metrics specified by the user. those 3 lines are in validation_step and training_step, but (apparently) have been forgottten here